scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -643,17 +643,27 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
               <td>{{gene.region_annotation}}</td>
               <td>
                 {% set n_primary_transcripts = [] %}
+                {% set hgnc_prim = namespace(primary=None) %}
```

Comment:
👀🤯😄
Elegant - meaning I had no idea one could do that! You will be the one updating this ... but I might try it somewhere else! :wink:

Reply:
Me neither, I've found it on Google! 😁